azurelinuxagent/ga/update.py

```diff
@@ -659,17 +661,9 @@ def __get_vmagent_update_status(self, protocol, goal_state_changed):
     def _report_status(self, exthandlers_handler):
         vm_agent_update_status = self.__get_vmagent_update_status(exthandlers_handler.protocol, self._processing_new_extensions_goal_state())
         # report_ext_handlers_status does its own error handling and returns None if an error occurred
-        #
```

Comment:
The first goal state is always Fabric, and now we ignore Fabric goal states from the HostGAPlugin, so 

`self._goal_state.extensions_goal_state.channel == GoalStateChannel.HostGAPlugin
`
would never be True. 

Now we do this in _try_update_goal_state(), using check_vm_settings_support()